src/java.base/share/classes/java/util/concurrent/CompletionStage.java

```diff
@@ -137,14 +140,20 @@
  *   }
  * }}</pre>
  *
- * <p>This interface does not define methods for initially creating,
+ * <p>The {@code CompletionStage} interface does not define methods for initially creating,
  * forcibly completing normally or exceptionally, probing completion
  * status or results, or awaiting completion of a stage.
  * Implementations of CompletionStage may provide means of achieving
  * such effects, as appropriate.  Method {@link #toCompletableFuture}
  * enables interoperability among different implementations of this
  * interface by providing a common conversion type.
  *
+ * <p>Memory consistency effects: Actions in a thread prior to the
+ * submission of a computation producing a {@code CompletionStage}
+ * <i>happen-before</i> that computation begins. And actions taken by
```

Comment:
Considering that it's the only occurrence of _happen(s)-before_ in this file, it makes sense to link it to its definition similarly to how it is done in other places.

Reply:
Thanks, done.